### DISCUSSION TITLE
Theme not being built properly when using GitHub Pages

### DISCUSSION THREAD

**Discussion by RC1242RC** [thuliteio/doks#811] [Q&A] [answered]
I am trying to get a simple docs site working via GitHub Pages. I've forked the example repo, and the only changes I've made are to update `baseurl` in `config.toml` and to change the `deploy-github.yml` to be triggered by `workflow_dispatch`. When I run the `deploy-github` action, it completes fine and then the standard GitHub `pages-build-and-deployment` also runs and completes. 

The produced website has all the content but none of the theming, almost as if GitHub has tried to build it with Jekyll (despite the `.nojekyll` file being present). Anyone know how I can fix this?

You can find the website at https://rc1242rc.github.io/gambit_gh_pages/

and the repo at https://github.com/RC1242RC/gambit_gh_pages

**Comment by h-enk** [comment, marked as answer]
When you view your website in e.g. Chrome and open [Chrome DevTools](https://developer.chrome.com/docs/devtools/) (F12) and next go to the Console tab, you'll notice in a number of Mixed Content errors: "This request has been blocked; the content must be served over HTTPS."

So, in your [config.toml](https://github.com/RC1242RC/gambit_gh_pages/blob/master/config/_default/config.toml#L1) update your `baseurl` to use `https` like so:

```toml
baseurl = "https://rc1242rc.github.io/gambit_gh_pages/"
# ..
```

**Reply by RC1242RC** [reply]
This solved the issue - thank you for your explanation and the quick reply!
